database/migrations/2021_08_20_083436_create_results_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateResultsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('results', function (Blueprint $table) {
+            $table->foreignId('sport_id')->constrained('sports');
```

Comment:
@jstarnate no need to specify the table `'sports'`, it would have been enough to have `constrained()`

Reply:
@PovilasKorop Noted :)